tedge_config/src/models/buffer.rs

```diff
@@ -0,0 +1,71 @@
+use serde::{Deserialize, Serialize};
+use std::convert::{TryFrom, TryInto};
+
+#[derive(Debug, Copy, Clone, PartialEq, Deserialize, Serialize)]
+pub struct Buffer(pub u16);
```

Comment:
The name is really confusing.

* `Buffer` sounds as the actual room for bytes, not as a size or a percentage.